docs/DeveloperGuide.md

```diff
@@ -155,6 +155,53 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### View Schedule Feature
+
+#### Implementation
+
+To implement `viewSchedule`, we made a new `FilteredList<Person>` object called `viewedPerson` in `ModelManager` to store the persons that we want to show.
```

Comment:
I think we can omit the names of the objects themselves, since we want to explain the higher level details first before moving to the details later on. Otherwise the reader may be confused at the start as to what `viewSchedule` and `viewedPerson` is. Maybe we can just explain in layman terms, something like "To allow users to view their contacts' schedules, we made a new `FilteredList<Person>` object in `ModelManager` to store ...